setup.py

```diff
@@ -21,7 +21,8 @@
     tests_require=['pytest'],
     entry_points={
         'console_scripts': [
-            'go2_sim = go2_simulation.go2_sim:main'
+            'go2_sim = go2_simulation.go2_sim:main',
```

Comment:
At this stage of dev, we can break the API, and rename go2_sim into go2_pybullet for instance (for a more consistent namimg)